src/pages/Profile.jsx

```diff
@@ -9,14 +9,12 @@ import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
 import CircularProgress from '@mui/material/CircularProgress';
 import Typography from '@mui/material/Typography';
-// Model Imports
```

Comment:
I'm a bit confused over the removal of these comments one line comments for separating imports (Line 12 and Line 14). Thought they were useful in organizing them within component jsx files?

Reply:
Yes. And we should be sure to spell it "Modal", not "Model" like above

Reply:
No model is correct, i.e. model for the data structure and schema. It's separate from the components that are modals that are used for the forms.

Reply:
I'll add them back in, but generally I think that comments like these just add noise to the file, and the info can be better communicated with good naming.

e.g. is this harder to understand:
```js
import { ClientModel } from '@models'
import { ContactsModel } from '@models'
```
than this:
```js
// Model imports
import { ClientModel } from '@models'
import { ContactsModel } from '@models'
```

Reply:
Fixed. The comments aren't identical because the profile component was refactored and given new imports.